integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/e2e/openstack/OpenStackTests.java

```diff
@@ -0,0 +1,222 @@
+package com.sequenceiq.it.cloudbreak.newway.testcase.e2e.openstack;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.clusterdefinition.responses.ClusterDefinitionV4ViewResponse;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.ResourceStatus;
+import com.sequenceiq.it.cloudbreak.newway.client.ClusterDefinitionTestClient;
+import com.sequenceiq.it.cloudbreak.newway.client.StackTestClient;
+import com.sequenceiq.it.cloudbreak.newway.cloud.HostGroupType;
+import com.sequenceiq.it.cloudbreak.newway.cloud.v2.openstack.OpenStackParameters;
+import com.sequenceiq.it.cloudbreak.newway.context.Description;
+import com.sequenceiq.it.cloudbreak.newway.context.SparklessTestContext;
+import com.sequenceiq.it.cloudbreak.newway.context.TestCaseDescription;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+import com.sequenceiq.it.cloudbreak.newway.dto.ClusterTestDto;
+import com.sequenceiq.it.cloudbreak.newway.dto.InstanceGroupTestDto;
+import com.sequenceiq.it.cloudbreak.newway.dto.clusterdefinition.ClusterDefinitionTestDto;
+import com.sequenceiq.it.cloudbreak.newway.dto.stack.StackTestDto;
+import com.sequenceiq.it.cloudbreak.newway.testcase.e2e.AbstractE2ETest;
+
+public class OpenStackTests extends AbstractE2ETest {
+
+    @Inject
+    private StackTestClient stackTestClient;
+
+    @Inject
+    private ClusterDefinitionTestClient clusterDefinitionTestClient;
+
+    @Test(dataProvider = "openStackHdpTestsDataProvider")
+    public void testOpenStackWithDefaulkHDPBlueprintsAndPrewarmedImagesWithoutGateWaySetup(
+            TestContext testContext,
+            String clusterDefinitionName,
+            String[] groups,
+            String scaleGroup,
+            @Description TestCaseDescription description) {
+
+        testContext
+                .given(ClusterTestDto.class)
+                .withClusterDefinitionName(clusterDefinitionName)
+                .given(StackTestDto.class)
+                .withInstanceGroupsEntity(getInstanceGroupTestDtos(testContext, groups))
+                .withCluster(ClusterTestDto.class.getSimpleName())
+                .when(stackTestClient.createV4())
```

Comment:
Will this test run with the `EDW-Analytics: Apache Hive 2 LLAP, Apache Zeppelin` cluster definition? That CD needs at least 3 worker nodes, and the e2e tests add 1 worker by default.